src/components/Button/FillButton.test.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import React from 'react';
+import { mount } from 'enzyme';
+import FillButton, { wrapColorsInGradient } from './FillButton';
+
+describe('FillButton component', () => {
+  it('should render button filled', () => {
+    const wrapper = mount(<FillButton />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  describe('Wrap colors in gradient', () => {
+    const testColor = '#fff';
+    it('should return single color (string)', () => {
+      expect(wrapColorsInGradient(testColor)).toEqual(testColor);
+    });
+    it('should return single color (array)', () => {
+      expect(wrapColorsInGradient([testColor])).toEqual(testColor);
+    });
+
+    it('should return a gradient', () => {
+      const colors = [testColor, '#000'];
```

Comment:
I think that this test case might be more maintainable if it's covered by a snapshot test of the `FillButton` component with the colors as a prop.

Reply:
I think it's OK to test it like this as it's directly testing the function that's generating the gradient. A snapshot would be also OK but I'm not expecting that this will change a lot.
If we notice that there are some changes to it we could change this to a snapshot.